tests/test_vjp.py

```diff
@@ -108,6 +108,18 @@ def test_zero_dy(self, dev, C):
         vjp = dev.compute_vjp(dy, jac)
         assert np.all(vjp == np.zeros([3]))
 
+    @pytest.mark.parametrize("C", [np.complex64, np.complex128])
+    def test_non_numpy_dy(self, dev, C):
```

Comment:
Should we also test this for the other frameworks too?

Reply:
I already added tests for all the ML frameworks in PR 239. Updating tests.yml was in TODOs :smile: I am glad there is now a separate PR for it.